source/elements/oneTBB/source/named_requirements/algorithms/par_reduce_func.rst

```diff
@@ -5,20 +5,20 @@
 ==================
 ParallelReduceFunc
 ==================
-**[req.parallel_reduce_body]**
+**[req.parallel_reduce_func]**
 
 A type `Func` satisfies `ParallelReduceFunc` if it meets the following requirements:
 
 -----------------------------------------------------------------------------------------------------
 
 **ParallelReduceFunc Requirements: Pseudo-Signature, Semantics**
 
-.. cpp:function:: Value Func::operator()(const Range& range, const Value& x) const
+.. cpp:function:: Value Func::operator()(const Range& range, Value&& x) const
 
-    Accumulates result for a subrange, starting with initial value ``x``.
-    ``Range`` type must meet the :doc:`Range requirements <range>`.
-    ``Value`` type must be the same as a corresponding template parameter for the
-    :doc:`parallel_reduce algorithm <../../algorithms/functions/parallel_reduce_func>` algorithm.
+    Accumulates values over a subrange, starting with the initial value ``x``.
```

Comment:
Probably, it is more correct to tell about the reduction here rather than accumulation as not any reduction necessarily accumulates. So, consider:

```suggestion
    Performs reduction of values over a subrange, starting with the initial value ``x``.
```